converter/src/main/java/de/ubleipzig/iiifproducer/converter/ConverterVersion3.java

```diff
@@ -0,0 +1,262 @@
+                    canvases.add(canvas);
+                }
+            });
+
+            List<MetadataVersion3> finalMetadata = metadataBuilderVersion3.execute();
+            ManifestVersion3 newManifest = buildManifest(viewId, canvases);
+
+            //build structures objects
+            final Optional<List<Structure>> structures = ofNullable(manifest.getStructures());
+            if (structures.isPresent()) {
+                final List<Structure> structs = structures.get();
+                final StructureBuilderVersion3 sbuilder = StructureBuilderVersion3.builder()
+                        .structures(structs)
+                        .viewId(viewId)
+                        .build();
+                sbuilder.buildIncrements();
+                List<Item> newStructures = sbuilder.build();
+                newManifest.setStructures(newStructures);
+            }
+
+            //set seeAlso
+            final Optional<String> finalURN = ofNullable(getURNfromFinalMetadata(finalMetadata));
+            @SuppressWarnings("unchecked")
+            List<String> related = (List<String>) manifest.getRelated();
+
+            final List<SeeAlso> seeAlso = setSeeAlso(viewId, finalURN.orElse(null), related);
+            newManifest.setSeeAlso(seeAlso);
+            newManifest.setMetadata(finalMetadata);
+
+            //set manifest label
+            final String manifestLabel = manifest.getLabel();
+            final Map<String, List<String>> manifestLabelMap = buildLabelMap(manifestLabel, NONE);
+            newManifest.setLabel(manifestLabelMap);
+
+            return newManifest;
+        } catch (IOException ex) {
+            throw new RuntimeException("Could not Convert Manifest", ex.getCause());
+        }
+    }
+
+    public ManifestVersion3 buildManifest(final String viewId, final List<CanvasVersion3> canvases) {
+
+        final List<String> contexts = new ArrayList<>();
+        contexts.add(WEB_ANNOTATION_CONTEXT);
+        contexts.add(IIIF_VERSION3_CONTEXT);
+
+        final String id = baseUrl + viewId + separator + manifestBase + ".json";
+
+        final List<String> behaviors = new ArrayList<>();
+        behaviors.add("paged");
+
+        final Map<String, List<String>> label = buildLabelMap("Attribution", ENGLISH);
+        final Map<String, List<String>> value = buildLabelMap(domainAttribution, ENGLISH);
+        final MetadataVersion3 requiredStatement = MetadataVersion3.builder()
+                .label(label)
+                .value(value)
+                .build();
+
+        final ManifestVersion3.Logo logo = ManifestVersion3.Logo.builder()
+                .id(domainLogo)
+                .type(IMAGE)
+                .build();
+
+        return ManifestVersion3.builder()
+                .context(contexts)
+                .id(id)
+                .type(MANIFEST)
+                .viewingDirection("left-to-right")
+                .behavior(behaviors)
+                .rights(domainLicense)
```

Comment:
`rights` cannot simply be a single default license. Public Domain Mark 1.0 is only applicable for manuscripts and historical prints when there are no other copyright restrictions. But there are several digitization projects at UBL that produce digital objects that are still in copyright. They often have multiple licenses linked in v2. The content of `rights` must reflect that.

To further complicate things, [`rights`](https://iiif.io/api/presentation/3.0/#rights) must be a single string value with either a creative commons or a rightsstatements.org URI; an array like v2 `license` is not allowed.

My suggestion would be to
* filter v2 `licenses` for the first entry that is either a creative commons or a rightsstatements.org license and use it for `rights`
* add _every_ license found in v2 `license` including the one that is used in `rights` as a HTML link at the end of the content of `requiredStatement`, e.g. `https://www.ub.uni-leipzig.de/recherche/katalog-information/nutzungshinweis-rechte-vorbehalten-freier-zugang/` becomes `<a href="https://www.ub.uni-leipzig.de/recherche/katalog-information/nutzungshinweis-rechte-vorbehalten-freier-zugang/">https://www.ub.uni-leipzig.de/recherche/katalog-information/nutzungshinweis-rechte-vorbehalten-freier-zugang/</a>`, either wrapped within a single `<div/>` or with necessary `<br/>`s for line breaks between all URIs and the main content of `requiredStatement`.
  * an alternative would be to reproduce those URIs whithin the manifest metadata but this would result in them not showing up in the rights panel in Mirador 3, for example.

Before you implement any of there suggestions: we are still waiting for feedback from Björn on what he thinks is best with respect to UBL manifests.

Reply:
implemented new required statement as suggested.  note that if license is included in attribution, then Mirador will show the value in both the "attribution" and "license" sections of the rights interface
![Screen Shot 2022-03-09 at 9 01 34 AM (3)](https://user-images.githubusercontent.com/5990301/157457040-837fbfa8-e01e-40c1-9f77-dbce333fe4ab.png)